auth.go

```diff
@@ -1011,6 +1011,19 @@ func (x *Central) UnArchiveIdentity(userId UserId) error {
 	return nil
 }
 
+func (x *Central) SetUserStats(userId UserId, action string) error {
+	e := x.userStore.SetUserStats(userId, action)
+	if e != nil {
+		x.Log.Warnf("SetUserStats failed: (%v), (%v)", userId, e)
```

Comment:
I would change this to error. It is important enough and affects our ability to accurately provide audit information.